gateway/Dockerfile

```diff
@@ -35,13 +28,12 @@ COPY ./gqlint ./gqlint
 COPY ./engine/crates ./engine/crates
 COPY ./workspace-hack ./workspace-hack
 
-RUN cargo build --release --target "$(rustc -vV | sed -n "s|host: ||p")" --bin grafbase-gateway &&\
-    mv "/grafbase/target/$(rustc -vV | sed -n "s|host: ||p")/release/grafbase-gateway" /grafbase/target/release/grafbase-gateway
+RUN cargo build --release --bin grafbase-gateway
 
 #
 # === Final image ===
 #
-FROM alpine:3.20
+FROM debian:bookworm
```

Comment:
Not that fussed if no, but could we use `bookworm-slim` as the base of the final image?

Reply:
do we want to use a specific version?

Reply:
Well the build image doesn't have latest debian, and points to bookworm, so we do too. This one will be supported for a _loooong_ time.